composer.json

```diff
@@ -28,9 +28,6 @@
     }
   },
   "config": {
-    "platform": {
```

Comment:
Please keep it and set it to php8.3

Otherwise if you are running php8.4 you will update to dependencies which are not compatible with 8.3 (though unlikely)

Reply:
can do, but since this is a dev dependency, do we need to be strict about the platform rules?

Reply:
It makes sure that the dependencies, even though dev are supporting 8.3 which transitively makes sure that Lychee support 8.3. If we do not have a strict requirement of 8.3, running this in 8.4 and end up with a dependency that is only 8.4 compatible, this incompatibility is directly translated to Lychee. Because running dev Lychee on 8.3 would not be possible, so we would have hard time figuring out which dependency is wrong.

More of a preventive measure than anything.

Reply:
alright, thanks for the clarification, already fixed